tools/dev/run_dev_server.sh

```diff
@@ -55,9 +55,9 @@ case $CONFIG in
   second-node)
       CONTEXTIONARY_URL=localhost:9999 \
       AUTHENTICATION_ANONYMOUS_ACCESS_ENABLED=true \
-      PERSISTENCE_DATA_PATH="${PERSISTENCE_DATA_PATH}-node2" \
+      PERSISTENCE_DATA_PATH="${PERSISTENCE_DATA_PATH:-"./data-node2"}" \
```

Comment:
Technically those two changes have nothing to do with this PR, but they add value anyway:

previously we would use a fixed volume and hostname for the second node. This made it impossible to run the PV dump of an actual node. Now, they're just defaults and we can override them. For example, I started a two node cluster with actual PVs like so:

```
CLUSTER_HOSTNAME=weaviate- PERSISTENCE_DATA_PATH=~/Downloads/weaviate-0 tools/dev/run_dev_server.sh local-no-modules 
CLUSTER_HOSTNAME=weaviate-1 PERSISTENCE_DATA_PATH=~/Downloads/weaviate-1 tools/dev/run_dev_server.sh second-node 
```